### DISCUSSION TITLE
B0B3F307-A97A-9012-08B4-C2F19B20AF8B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#11] [General]
# B0B3F307-A97A-9012-08B4-C2F19B20AF8B

Week 33, 2022: NameExoWorlds 2022 has been Announced!

https://celestia.mobi/article?guide=B0B3F307-A97A-9012-08B4-C2F19B20AF8B

**Comment by DaveBowman2001** [comment]
![Untitled](https://user-images.githubusercontent.com/95486841/226804535-ec3b25f1-60df-4124-ae0b-4dae9c00fd2a.jpg)
_Montage of every **NameExoWorlds (2022)** candidate on Celestia (screenshot by [*Gurren Lagann*](https://celestia.mobi/resources/itemsByAuthor?author=Gurren%20Lagann))_

**Comment by DaveBowman2001** [comment]
UPDATE (230122)
> As of 2023-Mar-19, all _**NameExoWorlds (2022)**_ candidates have add-ons available for download in Celestia! Here are the links below:
- [**GJ 1214**](https://celestia.mobi/resources/item?item=DDCEF263-FEA3-083D-6439-DC0D3043DA5E) by **Edasich**
- [**GJ 3470**](https://celestia.mobi/resources/item?item=E7BE253E-3EF2-42B2-9BA8-4E7FE36C4DEE) by **Gurren Lagann**
- [**GJ 4332**](https://celestia.mobi/resources/item?item=71361762-3625-4E90-A8C6-2BA14BE3B665) by **Trolligi**
- [**Gliese 367**](https://celestia.mobi/resources/item?item=6E0E0EFB-AEE1-42BC-9521-1E6684510FE0) by **SevenSpheres**
- [**Gliese 436**](https://celestia.mobi/resources/item?item=96FC5047-E5C2-ECEF-ADAD-A1BB451A6A6D) by **MrSpace43**
- [**Gliese 486**](https://celestia.mobi/resources/item?item=249C5647-7260-4687-B647-30EA66362D7F) by **Gurren Lagann**
- [**HAT-P-12**](https://celestia.mobi/resources/item?item=79B42FA9-A6BA-DB26-A6CE-BBC04388693A) by **Edasich**
- [**HAT-P-26**](https://celestia.mobi/resources/item?item=759EA5F2-6C91-D2D5-F798-F4EF4582CBC0) by **Edasich**
- [**HATS-72**](https://celestia.mobi/resources/item?item=F05A8DD6-70C3-49CB-90D8-E79EF01C05A1) by **SolHorizon**
- [**HD 95086**](https://celestia.mobi/resources/item?item=38117E90-9DC1-4D31-806C-E8EAF756154E) by **Gurren Lagann**
- [**HIP 65426**](https://celestia.mobi/resources/item?item=4D5862DC-600E-46FD-B28F-3821B2C98BA1) by **John van Vliet**, et. al.
- [**LHS 3844**](https://celestia.mobi/resources/item?item=5E931776-08C7-D493-2DB8-E43C9C279D34) by **trappistplanets**
- [**LTT 9779**](https://celestia.mobi/resources/item?item=36EE8D3E-C3D7-433F-9834-CF25A1E7B625) by **Gurren Lagann**
- [**WASP-19**](https://celestia.mobi/resources/item?item=98A23DF9-31D5-4C44-B3B7-36431E06EB8F) by **MrSpace43**
- [**WASP-43**](https://celestia.mobi/resources/item?item=41E9B7A2-006D-4C5C-ADDC-B71E3DF857A6) by **MrSpace43**
- [**WASP-63**](https://celestia.mobi/resources/item?item=124A7FC9-6682-4D01-9667-D84636D4C770) by **Gurren Lagann**
- [**WASP-69**](https://celestia.mobi/resources/item?item=37CEC0A8-D147-4E8D-9833-6D0D1DB978AB) by **Gurren Lagann**
- [**WASP-121**](https://celestia.mobi/resources/item?item=EAE2E44F-CB01-4370-AC16-632E5A5A9D5D) by **Pan-STARRS**
- [**WASP-166**](https://celestia.mobi/resources/item?item=2C58AD2A-443D-43F8-8344-C519B6223377) by **Gurren Lagann**
- [**WD 0806-661**](https://celestia.mobi/resources/item?item=D1EB6480-6320-42A7-9842-C1785FB4F41C) by **Gurren Lagann**
